controllers/v1/observationsController.js

```diff
@@ -819,7 +819,11 @@ module.exports = class Observations extends Abstract {
 
         let programQueryObject = {
           _id: observationDocument.programId,
-          status: "active"
+          status: messageConstants.common.ACTIVE_STATUS,
+          tenantId: req.userDetails.tenantData.tenantId,
+          'scope.organizations': {
```

Comment:
@borkarsaish65, as discussed with @priyanka-TL, we should not have this check here. Since organizations can be an optional factor in a tenant, it would be a conflict here, since its a hardcoded filter.

Solution - Remove org check here. Implement targeting logic check here so that if I am not targeted for a program or solution there is no way I can consume the same.